en/faq.md

```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
 
 [[toc]]
 
-## 站点使用相关
+## Station usage related
 
-### 收不到验证邮件？
+### Can't receive verify email？
 
-请先检查你的邮箱的垃圾箱。如果垃圾箱中也没有验证邮件，尝试再发送一次验证邮件。
+Please check your email's trash can at first. If you can't find verify email in trash can, please try send verify email again.
 
-如果仍然收不到验证邮件，请检查你的账号绑定的邮箱是否正确，如果不正确则将其更改为正确的邮箱。
+If you still can't receive verify email, please check your email that bind your account is correct, if incorrect, please change it to your correct email.
 
-如果你确定你绑定的邮箱正确，并且验证邮件不在你的邮箱的垃圾箱里，请加入我们的 [官方用户交流群](/user-group.html)，联系站点管理员处理。
+If you sure your email that bind your account is correct, and verify email can't find in your trash can, please join our [User Group](/user-group.html), contact staff to process.
 
-### 我的角色被别人占用了，我可以要回来吗？
+### My profile are have by others, I can get it back?
 
-不可以。重名的几率很大，随意更改角色所有者对双方都不公平，所以不管你是名声多大的主播还是某个达官贵人家的孩子，我们都不会提供更改角色所有者的服务。
+No. The chance of having the same name is very high, change players' owner is unfair to both users, so whether you are an anchor or a rich people, we don't provide the service that change players' owner.
```

Comment:
No. The chance of having the same name is very high, change profiles' owner is unfair to both users, so we won't change profiles' owner for everyone.